Snippets/end.Checks.groovy

```diff
@@ -366,7 +366,7 @@ def generalCheck(String codeInvalid, String sheet, List fields, String message,
 		def valid = validator.call(['value':values.join('+'), 'sheet':sheet, 'field':fields.join('+')])
 		
 		if (!valid) {
-			_log.error("${codeInvalid}: The values ${values} for the fields ${fields} ${message} [${sheet}]${id}")
+			_log.error("${codeInvalid}: The values ${values} for the fields ${fields} ${message} [${sheet}]")
```

Comment:
@annat2022 `${id}` can be safely removed here. It was most likely an oversight by me when I copied [this line](https://github.com/wetransform/uba-end-transformations/blame/d4d4613ba50dab4a335e80f7016aec4758378e7c/Snippets/end.Checks.groovy#L402).

Reply:
That's what I assumed, just wanted to make sure.